### ISSUE TITLE
Feature Request: move caret to current word being read

### ISSUE THREAD

**Issue by TechHorseG** [nvaccess/nvda#10817]
Win10 1909 / Version: 2019.2.1 / portable

Hello, this request is that when reading text with a caret, NVDA moves the cursor to the  ccurrent word being read.

I might use "say all" to check the spelling of something I have written, but keep getting caught out by the fact that the caret might be somewhere else and not at the point I stopped reading because I have just heard a mispronounced word and so need to correct the spelling.

I'll press ctrl+left to move backk to the word I had just heard, only to realise I am somewhere else and so need to work out where I am in relation to the target word.

Thank you, hope this is possible!

**Comment by Adriani90** [comment]
Please fill the issue template with all the details so we cam reproduce this issue. From your description I cannot reproduce it. Nvda always moves the carret to the word being read.x

**Comment by TechHorseG** [comment]
Ah sorry. Text files in Notepad are an example.. The cursor (generally) moves to the beginning of each  line., but stays there.
